<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::WheelFocus</enum>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="src.qrc">
    <normaloff>:/src/logo2.png</normaloff>:/src/logo2.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	rgb(90,90,90)
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QMenuBar" name="menubar">
     <widget class="QMenu" name="menuAbout">
      <property name="title">
       <string>Help</string>
      </property>
      <addaction name="actionHelp"/>
      <addaction name="actionAbout_qpImage"/>
     </widget>
     <addaction name="menuAbout"/>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="2">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="prevBtn">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="src.qrc">
           <normaloff>:/src/previous90.png</normaloff>:/src/previous90.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="nextBtn">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="src.qrc">
           <normaloff>:/src/icons8-next-page-90.png</normaloff>:/src/icons8-next-page-90.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1" rowspan="3">
        <widget class="QGraphicsView" name="graphicsView">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>8</width>
     <height>17</height>
    </rect>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
  </widget>
  <action name="actionAbout_qpImage">
   <property name="text">
    <string>About qpImage</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
